helpers/travelBooking.helper.js

```diff
@@ -99,7 +99,7 @@ export async function bookTravelForMumukshus(date, mumukshuGroup, t, user) {
 
   const bookings = await TravelDb.findAll({
     where: {
-      type: TRAVEL_TYPE_SINGLE,
+      type: TRAVEL_TYPE_REGULAR,
```

Comment:
this will cause issues with existing travel bookings with type as "single", right?